src/server/db/models/github-user.js

```diff
@@ -9,7 +9,7 @@ import db from '../index';
  *   updated_at: update date
  *   last_login_at: last login date
  */
-export const GitHubUser = db.Model.extend({
+db.model('GitHubUser', {
```

Comment:
Or maybe - to push this idea to the edge, make models that export just a schema, and db module will register them:

`/models/github-user.js`:
```js
export default {
  tableName: 'github_user',
  hasTimestamps: true
}
```
`/index.js`:
```
import * as models from `./models`;

// db module code

db.model('GitHubUser, models.GitHubUser);

export default db;
```

But this makes model module almost empty and useless (as the whole schema is defined in migration file anyway)...